shuup_tests/functional/test_discount_taxing.py

```diff
@@ -205,7 +209,7 @@ def test_3prods_services_2discounts(taxes):
             'Tax-C   0.50   45.333333333   22.666666667   68.000000000',
             'Tax-A   0.25   16.320000000    4.080000000   20.400000000',
             'Tax-B   0.20   88.541666667   17.708333333  106.250000000',
-            'Total   0.30  150.190000000   44.455000000  194.650000000']
+            'Total   0.30  150.180000000   44.455000000  194.650000000']
```

Comment:
Well not sure how this should be. This basically happens in this case the tax_amount in source used to have over 9 decimals. And when ensuring tax amount to 9 decimals we go 1 cent. But without ensuring the 9 decimals source line taxes does not match order line taxes and will cause some other places to fail.